src/styles/index.scss

```diff
@@ -1,3 +1,71 @@
-body {
+@import 'variables';
+
+* {
   margin: 0;
+  padding: 0;
+}
+
+body {
+  font-family: $calendar-font;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+}
+
+.calendar {
+  gap: $calendar-gap;
+  display: flex;
+  align-items: center;
+  justify-content: flex-start;
+  flex-wrap: wrap;
+  padding: 10px;
+  width: calc(($calendar-gap + $calendar-width) * 7);
```

Comment:
The width calculation for the .calendar class should include the width of the borders as well. Adjust the calculation to account for this.